src/lxc/cgroups/cgfsng.c

```diff
@@ -329,15 +591,18 @@ bool handle_cpuset_hierarchy(struct hierarchy *h, char *cgname)
 		return false;
 	}
 
+	/* Make sure any isolated cpus are removed from cpuset.cpus. */
+	if (!set_cpuset(cgpath, v == '1'))
```

Comment:
Given the comment, it sounds like set_cpuset() should be renamed to something like filter_cpuset() or remove_isol_cpuset()?